superset/assets/src/explore/controls.jsx

```diff
@@ -1514,6 +1514,14 @@ export const controls = {
     'point in time'),
   },
 
+  chart_interactivity:{
+    type: 'CheckboxControl',
+    label: t('Interactivity'),
+    renderTrigger: true,
+    default: false,
+    description: t('This is used for enable/disable chart click event'),
```

Comment:
we may need to add translation text for various language in message.po. we will create a US for the same